### PR TITLE
OCaml 4.13.0 release

### PR DESCRIPTION
**Pull request by Octachron**
This is the usual release update for OCaml 4.13.0: the version 4.13 of the manual, and the new information page for the release. I will update the PR once I know the actual release date. I have also opened an issue on ocaml/ood#85 to see if we can improve the process on the current run.

### CODE REVIEW

File: site/releases/index.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ See also the [install](/docs/install.html) page for instructions on
 installing OCaml by other means, such as the OPAM package manager and
 platform specific package managers.
 
-* OCaml [4.13.0](4.13.0.html), [Download](https://github.com/ocaml/ocaml/archive/4.13.0.tar.gz), released Sep ??, 2021.
+* OCaml [4.13.0](4.13.0.html), [Download](https://github.com/ocaml/ocaml/archive/4.13.0.tar.gz), released Sep 24, 2021.
+* OCaml [4.12.1](4.13.0.html), [Download](https://github.com/ocaml/ocaml/archive/4.13.0.tar.gz), released Sep 24, 2021.
```
Comment (pmetzger) [inline site/releases/index.md]: This looks wrong, shouldn't that refer to `4.12.1.html`?

Comment (pmetzger) [reply site/releases/index.md]: (And also 4.12.1.tar.gz of course.)

Comment (Octachron) [reply site/releases/index.md]: Indeed, fixed, thanks!

**Comment by Octachron** [comment]
This PR can be merged after https://github.com/ocaml/opam-repository/pull/19636 .

**Comment by pmetzger** [comment]
So presumably this should be merged ASAP since the opam update happened.

**Comment by Octachron** [comment]
Let's merge and update if there is an issue at hand.

**Pull request merged**